src/components/MenuBar/ButtonBar.js

```diff
@@ -80,13 +93,25 @@ function ButtonBar({ isMobile }) {
 				<PoppinLink to={link.to} key={`nav-${index}`}>
 					{link.name}
 				</PoppinLink>)}
-			{
+
+			{Date.now() < hothStart.getTime() ?
 				<BorderLink
-					disabled={Date.now() < applicationOpen.getTime() || Date.now() > applyDeadline.getTime()}
+					disabled={
+						Date.now() < applicationOpen.getTime() ||
+						Date.now() > applyDeadline.getTime()
+					}
 					href={'https://forms.gle/VMhdCzMov8RvGUfP8'}
-					target='_blank'
+					target="_blank"
 				>
 					Apply
+				</BorderLink>			 :
+				<BorderLink
```

Comment:
After HOTH ends, this `Submit` button will show as a disabled button, which is not an issue but is that functionality you want? Imo, it might be weird to have a disabled `Submit` button for the rest of the year when HOTH is not running, but we can always fix that later.